src/turbolinks/controller.coffee

```diff
@@ -179,8 +179,8 @@ class Turbolinks.Controller
   notifyApplicationAfterRender: ->
     Turbolinks.dispatch("turbolinks:render")
 
-  notifyApplicationAfterPageLoad: (timing = {}) ->
-    Turbolinks.dispatch("turbolinks:load", data: { url: @location.absoluteURL, timing })
+  notifyApplicationAfterPageLoad: (timing = {}, initial = false) ->
```

Comment:
Instead of positional parameters, let's use an object so we can name its keys. This lets us avoid the awkward `@notifyApplicationAfterPageLoad(undefined, true)` construct.

```coffee
notifyApplicationAfterPageLoad: ({timing, initialPageLoad} = {}) ->
  Turbolinks.dispatch("turbolinks:load", data: { url: @location.absoluteURL, timing, initialPageLoad })
```

Now we can write this:
```coffee
@notifyApplicationAfterPageLoad(initialPageLoad: true)
```